monero-outreach-docs/translations/zh-hans/investor_resources/investor_resources_zh-hans.md

```diff
@@ -0,0 +1 @@
+Investor Resources
投资者资源

Services and platforms available for Monero investments.
门罗币投资的服务与平台

DISCLAIMER: For your safety, it is recommended that you do your own research before using any third party for managing your Monero. Inclusion on this page does not imply any manner of recommendation or endorsement.
免责声明:为了你的安全，建议您在使用任何第三方管理你的门罗币之前，先进行自己的研究。本网页的内容并不代表任何推荐或认可。
 
Decentralized Exchanges 去中心化交易所

Decentralized or P2P (Peer-to-Peer) exchanges are the best way to buy/sell Monero by yourself while keeping your privacy intact.
去中心化交易所或者点对点交易所是您自己买卖门罗币的最好方式，同时还可以保持您的隐私。

Centralized, non-KYC Exchanges 中心化交易所，非身份认证交易所
These exchanges do not require ID verification at least for certain amounts.

KYC Exchanges   身份认证交易所
These KYC (Know Your Customer) compliant exchanges will require identification before you can participate.
这些采用身份认证的交易所要求身份证明才能加入。

Custodian Services 客户服务
If you are not confident about the security of your keys against theft or data loss, a custodian service will hold your Monero for you.
如果你对你的密钥的安全性没有信心，以防被盗或数据丢失，托管服务将为你保管你的门罗币。

Investment Vehicles  投资工具
Investment vehicles allow investors to gain exposure to Monero price movements without having to purchase any actual XMR, it is primarily for speculation on the price.

投资工具允许投资者在不需要购买任何实际门罗币的情况下获得门罗币价格波动的情况，主要是为了对价格进行推测。

```

Comment:
This section in missing in the 'en' website-sections but will add it today. Anyway this should be moved to a different folder too:

- Move to folder _/zh-hans/website-sections_

**The files name is okay, I will add the 'en' version with the same name.**

Reply:
Done.